nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/controller-service/controller-service-table/controller-service-table.component.html

```diff
@@ -31,13 +31,11 @@
                 <td mat-cell *matCellDef="let item">
                     @if (canRead(item)) {
                         <div class="flex items-center gap-x-3">
-                            <div class="pointer fa fa-book" title="Usage"></div>
-                            <!--
-              nifiTooltip dynamically inserts the tooltip component
-              into the dom. the dom placement caused shifting of icons
-              because of the gap between items. simple solution is to
-              just wrap the target.
-              -->
+                            <div
+                                class="pointer fa fa-book"
+                                (click)="viewDocumentationClicked(item, $event)"
+                                title="View Documentation"></div>
```

Comment:
title versus tooltip?

Reply:
We could look at other options when considering our strategy for tooltips throughout. We still need to address all the info icons. But thus far, all buttons and icons in table listings have used `title`. Here I only updated the value.